packages/react/src/components/Combobox/Combobox.tsx

```diff
@@ -439,11 +652,36 @@ const Combobox = forwardRef<HTMLDivElement, ComboboxProps>(
           // there's already keyboard handlers to open the listbox on the input element
           onClick={handleInputClick}
         >
+          {multiselect &&
+            selectedValues.map((value, index) => {
+              const refCallback = (elem: HTMLButtonElement | null) => {
+                if (elem) {
+                  pillsRef.current[index] = elem;
+                } else {
+                  pillsRef.current.splice(index, 1);
+                }
+              };
+
+              const handleClick = () =>
+                handleRemove(pillsRef.current[index], value);
+
+              return (
+                <ComboboxPill
+                  ref={refCallback}
+                  key={value}
+                  value={value}
+                  removeValueAriaLabel={removeValueAriaLabel}
```

Comment:
A couple of things with this property. We don't usually call things `xxxAriaLabel`. Secondly, the way this is written is assuming [subject-verb-object](https://en.wikipedia.org/wiki/Subject%E2%80%93verb%E2%80%93object_word_order) word order by concatenating the label with the string. This might make it impossible to translate correctly in some languages.

My recommendation here would be to set an optional property `removeOptionLabel` on `ComboboxOption` that would allow for the user to override the default, e.g. we would concatenate `remove {label}` if this property wasn't set, otherwise we'd use the value from that property. This would allow for things to be translated correctly in product land.